SysEng/dataInputFormat.md

```diff
@@ -0,0 +1,63 @@
+## Finalized Input Data format
+
+The data for the pressure drop calculator will be input in the following manner:
+
+### Python file format
+```python
+[
+  CID, 
+  ['deltaX', 'deltaZ','radius','angle'],
+  ['insideArea','massFlow','D_h'],
+  ['valveCoefficient','specificGravity','N'],
+  ['upstreamArea','downstreamArea','contractionType (a or c)',['contractionParameters']]
+  ['rho', 'mu', 'T'],
+  ['rey', 'f','kt', 'deltaP']
+]
+```
+
+### .json/JavaScript file format
+```javascript
+{
+  "componentID": CID,
+  "distanceValues": [
+    {
+      "lengthChange" : deltaX,
+      "altitudeChange" : deltaZ,
+      "bendRadius" : radius,
+      "bendAngle" : angle
+    }
+  ],
+  "miscInfo" : [
+    {
+      "valveCoefficient" : valveCoefficient,
+      "contractExpandUpstream" : upsteamArea,
+      "contractExpandDownstream" : downstreamArea,
+      "contractionAngledOrCurved" : expandType,
+      "contractionParameters" : [
+        {
+          "angle" : contractAngle,
+          "downstreamDiameter" : contractDiameter,
+          "downstreamRadiusOfCurvature" : contractRadius
+        }
+      ]
+    }
+  ],
```

Comment:
the .json file type does not match up with the formatting of the python file type. Valves should be it's own category.